src/io/pti.jl

```diff
@@ -943,4 +943,744 @@ function _populate_defaults!(data::Dict)
+        sbase = data["baseMVA"]
+
+        if type == "2W"
+            # Get default owner
+            bus_i = transformer["f_bus"]
+            owner = bus_owner[bus_i]
+                        
+            # Get the source data
+            source_type = data["source_type"]
+
+            # Convert to two winding transformer
+            psse_comp = _pm2psse_2w_tran(transformer, owner, sbase, source_type)
+            
+            lines = [
+                "TRANSFORMER",
+                "TRANSFORMER TWO-WINDING LINE 1",
+                "TRANSFORMER TWO-WINDING LINE 2",
+                "TRANSFORMER TWO-WINDING LINE 3",
+                ]
+                
+        elseif type == "3W"
+            # Get default owner
+            bus_i = transformer["w1"]["f_bus"]
+            owner = bus_owner[bus_i]
+            
+            # Convert to Three winding transformer
+            psse_comp = _pm2psse_3w_tran(transformer, owner, sbase)
+            
+            lines = [
+                "TRANSFORMER",
+                "TRANSFORMER THREE-WINDING LINE 1",
+                "TRANSFORMER THREE-WINDING LINE 2",
+                "TRANSFORMER THREE-WINDING LINE 3",
+                "TRANSFORMER THREE-WINDING LINE 4",
+                ]
+        end
+        
+        for (psse_part, line) in zip(psse_comp, lines)
+            _print_pti_str(io, psse_comp, _pti_dtypes[line])
+        end
+    end
+
+    println(io, "0 / END OF TRANSFORMER DATA, BEGIN AREA DATA")
+
+    # Area Interchange
+    if haskey(data, "area interchange") 
+        for (_, area) in sort(data["area interchange"], by = (x) -> parse(Int64, x))
+            # Get Dict in a PSSE way and print it
+            psse_comp = _pm2psse_area_interchange(area)
+            _print_pti_str(io, psse_comp, _pti_dtypes["AREA INTERCHANGE"])
+        end
+    else
+        Memento.warn(_LOGGER, string("Skipping AREA INTERCHANGE data because it does not found"))
+    end
+    
+    println(io, "0 / END OF AREA DATA, BEGIN TWO-TERMINAL DC DATA")
+
+    # TODO : See how PM converts the DC line and do the oposite
+    Memento.warn(_LOGGER, string("Export TWO-TERMINAL DC data is not yet supported"))
+    
+    println(io, "0 / END OF TWO-TERMINAL DC DATA, BEGIN VOLTAGE SOURCE CONVERTER DATA")
+    
+    # TODO : See how PM converts the DC line and do the oposite
+    Memento.warn(_LOGGER, string("Export VSC data is not yet supported"))
+    
+    println(io, "0 / END OF VOLTAGE SOURCE CONVERTER DATA, BEGIN IMPEDANCE CORRECTION DATA")
+
+    Memento.warn(_LOGGER, string("Export IMPEDANCE CORRECTION data is not yet supported"))
+    
+    println(io, "0 / END OF IMPEDANCE CORRECTION DATA, BEGIN MULTI-TERMINAL DC DATA")
+    
+    Memento.warn(_LOGGER, string("Export MULTI-TERMINAL DC data is not yet supported"))
+    
+    println(io, "0 / END OF MULTI-TERMINAL DC DATA, BEGIN MULTI-SECTION LINE DATA")
+    
+    Memento.warn(_LOGGER, string("Export MULTI-SECTION data is not yet supported"))
```

Comment:
I would only warn if the data contains the key that is associated with `import_all=true` and the data in there is not empty.  The message should say "not yet supported by the export_pti function"

Reply:
Solved at the top of the function

Reply:
I didn't see it, can you clarify where this is happening?

Reply:
Sorry for the mess. It is happenning [here](https://github.com/lanl-ansi/PowerModels.jl/pull/753/commits/043ec834f90e996db76f12764efec643a2c256ed#diff-8eaa62afd9f02012047d607fc11deb5fc6d076a155ecf49c2ba89625666653afR1018)